### DISCUSSION TITLE
What's the right choice Quasar CLI vs UMD?

### DISCUSSION THREAD

**Discussion by mbnoimi** [quasarframework/quasar#14686] [CLI - Capacitor / Cordova modes]
Hi,
I'm planning to create my project in less headache but I'm really confused. So may you please help me to take the right decision?

I'll build my project using Laravel 9 + Inertia + Vue 3 it's a fantastic cocktail for a webapp but it's not suitable for mobile & desktop deployment because of Inertia. For that I'll use Quasar but with this Laravel cocktail it's impossible to use Quasar CLI. Instead I've to use Quasar UMD which is missing a lot of awesome features I'll definitely need them (ex, Solid RTL support, Desktop App and One codebase)

Inertia is very important to me because it gives me the ability to use Laravel's router instead of Vue router. I don't want to build an API and dive in the complexity of client-side routing. Unfortunately Quasar CLI doesn't work with Inertia (as I read in some discussions here) for that the only available choice is Quasar UMD. **BUT** I wonder does it worth to drop Inertia in favor of Quasar CLI!?

Which choice do you advice me as you have the experience with Quasar?

1. Laravel 9 + Inertia +  Quasar UMD
2. Laravel 9 + Quasar CLI

Do you think there are 3<sup>rd</sup> or 4<sup>th</sup> available choices to my scenario?

Thanks in advance.

**Comment by alihardan** [comment]
Hi @mbnoimi, I'm in the same question as you were. What did you do about this?

**Reply by mbnoimi** [reply]
> What did you do about this?

I dropped Inertia (later I dropped Laravel itself :) ) because Quasar CLI is really worth it.
I changed my tech stack dramatically to be as following:

**For Webapps:**
- Backend: Go
- Frontend: Quasar CLI (SPA or SSR)

**For Websites:**
- Hugo OR Astro (with Quasar UMD)
